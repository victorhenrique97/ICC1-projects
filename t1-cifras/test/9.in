1660
LOREM IPSUM DOLOR SIT AMET, CONSECTETUR ADIPISCING ELIT. MORBI FERMENTUM QUIS ENIM NEC ULTRICES. CLASS APTENT TACITI SOCIOSQU AD LITORA TORQUENT PER CONUBIA NOSTRA, PER INCEPTOS HIMENAEOS. NAM DOLOR EROS, VULPUTATE UT RISUS EGET, HENDRERIT BIBENDUM ARCU. NULLAM AC MAGNA VEL ANTE CURSUS MATTIS A AC NUNC. IN HAC HABITASSE PLATEA DICTUMST. ETIAM LOBORTIS EX TELLUS, EGET PORTTITOR RISUS PORTTITOR IN. MORBI AT CONSEQUAT LOREM. VIVAMUS A PELLENTESQUE DOLOR. SUSPENDISSE VEL PURUS ULTRICIES ERAT DICTUM FACILISIS. MORBI NEC CONDIMENTUM SEM. PRAESENT SEM RISUS, SCELERISQUE AC NUNC UT, DICTUM VENENATIS SAPIEN. DONEC NON SODALES NULLA, UT DAPIBUS JUSTO. IN NEC ULTRICIES MI. INTEGER QUIS EROS EGET MAGNA LOBORTIS TINCIDUNT AC QUIS NUNC. FUSCE RUTRUM MAURIS ORCI, UT RUTRUM EROS RUTRUM EU. INTEGER EGET JUSTO UT ELIT SODALES ORNARE ALIQUET A METUS. DONEC FINIBUS TINCIDUNT ERAT, SIT AMET LOBORTIS QUAM. MAECENAS IN LIBERO ORCI. UT NON ULTRICES ENIM. SED VITAE FACILISIS ARCU, IN PULVINAR LECTUS. NULLAM ODIO NISL, SEMPER MOLESTIE LOREM NEC, ALIQUAM HENDRERIT ENIM. PHASELLUS NON FELIS AUCTOR, FRINGILLA PURUS MAXIMUS, PHARETRA ERAT. MORBI METUS DUI, VEHICULA SIT AMET PULVINAR EGET, ULTRICES ID LIBERO. MAECENAS NIBH ARCU, VESTIBULUM PULVINAR NISI CONSEQUAT, DAPIBUS DIGNISSIM ENIM. FUSCE A ANTE CONSEQUAT, TEMPUS LACUS QUIS, PRETIUM NEQUE. SUSPENDISSE SIT AMET SODALES DIAM. FUSCE ET TORTOR NON ORCI SOLLICITUDIN IACULIS. PHASELLUS PORTA LECTUS SED ENIM CONSEQUAT, ID CURSUS DUI RHONCUS. AENEAN GRAVIDA ARCU RISUS, AT MATTIS NIBH VEHICULA QUIS. PRAESENT TORTOR SAPIEN, TINCIDUNT VOLUTPAT BLANDIT IN, VIVERRA VEL ENIM. CRAS NEQUE TELLUS, CURSUS EGET PORTTITOR QUIS.
2
10
FMQVIALTMA
