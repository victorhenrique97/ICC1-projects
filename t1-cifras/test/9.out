QAHZU IALGM IABJZ STM MMJF, SJVSPVFEYGH VLIABECNZW ZTIE. FARGU VZZMPGFUR CKDA EYBY NJO KGBRTVQS. HXQNA AAMQNY FQXQTT LACNAILC AO EUTTDQ OWRBNQNY BUM KOYNNIF ZENBRL, IQR NZSZXTZL TIRQDVMOD. GMM IABJZ ECHE, VZXFPBAEX GT WUIPA ERXF, HJZTMMRTM NIGQDYCM LKOU. SGBGIM LV YALZQ QML LGFE HGHNCS XTFTNE Q VK NFGO. IS TQX PAMBFAXEU KTAEXM DNOJPUSE. XFIFY BJJOCMUS JJ JZTLFL, QGJF FJZTEBFOW DYNCS AHDTYUJJZ IY. FARGU QO KOYLQQZMJ GWRPF. HIAMCPA A AXXLJZJZAQFX POQAH. NCSAXZDNEIZ DEW IGRZE KGBRTVUEX QHVB DTVFUR RQXQLTLUS. RAHWQ NPV OOSPYHMNENY SJY. FMIEDXZT XQC MQSFL, ECJXUMQSBNQ AH ZKIK UE, WUCYGC QMNPGMTNE IVXIPG. POSQS IWN DHPAQQI ICLWT, GT IMFDJUD CGSYA. YI VEN NXTWUSDMS XB. UNYQWZZ QFBE EWAI ZOEE FMGSM BJJOCMUS YUDXQDFGF AH CKDA NFGO. FZESZ ZUEKGM RMKMQS ZKOI, ZF HPBRFF QRTE HPBRFF QU. NZJZOEC XSEY VKNBO FM QLNF IJLAWXE OWZQMM AWBCUJF Q HMTFL. POSQS AQNTUGS YUDXQDFGF EWMJ, NQT LFQT QARJZTTL CUFY. CVMCPGMS NZ BDJECH ARHU. KO VOY NXTWUSZA EYBY. SJP LDBAP YMCNXYNQS LKOU, NZ FPTVTGMR QQSOCS. YNXLFY EYQO YBEL, XQCKMR XHXEXFYZ TOCXY NJO, QGQQFTY HJZTMMRTM QNNY. FCISPEXUX ZEI NEWBE AZOJJZ, FCBZGNXBV XUCNE MFJYHCS, AAMRJFHV MRLM. YOWNY HMTFL PUN, HUCQCFEM SNF QHMT ANXVNZQM MGPM, GLYDYXMS TW XIGQHJ. UAPVQNFE DDJH LKOU, AQIOQBFEGM UGBQQNLK ZIXU SJVSPJGAY, PQKQBFL PILZYNAIX XZIR. RKNKE L TZTJ OEIAEBNMT, YQCKCS WTOUX CKDA, PCXFIZY DZYUP. LGSUQDYQSDX EIY MCZB SZWMLJE TDIM. QNECJ QJ OWREHD NTZ EMKI DHXLNOYOCDTG UAHGBDA. PSTEEQXKN XOCMM LJOJPA SPW QNNY SJVSPJGAY, UT XCRDNE DZU HCWNNNE. AJZUVV GCTHIIM QMKU CBEUX, MJ HITEBE NNNX QMHTVGLF CKDA. PCTQSJZJ OWREHD SFBYZV, TTGOIIGDO DOWNFPFF RGINOBF IS, HYQMRCT HEQ QDDU. CCTE NJCKZ BEWEGS, HGHNCS PZQT UAHOBIEHD QZUI.
